convert.py

```diff
@@ -87,7 +89,7 @@ def parse() -> argparse.Namespace:
         help="Path to directory to write the log files",
         required=False,
     )
-    return parser.parse_args()
+    return parser.parse_args(argv[1:])
```

Comment:
We don´t, this was a leftover from my first attempt testing the cli using subprocess, where the first arg would have been "convert.py". Now we don´t need it anymore, I removed it from all cli tests and set it to arv (instead of argv[0]) in all cli scripts